exporter/diagnostic_data_collector.go

```diff
@@ -36,25 +37,38 @@ func (d *diagnosticDataCollector) Collect(ch chan<- prometheus.Metric) {
 		return
 	}
 
-	for _, metric := range d.makeMetrics("dd", m) {
+	for _, metric := range d.makeMetrics("", m, nil) {
```

Comment:
don't we need prefix?

Reply:
No. It will receive the prefix while going deep in the tree structure. The initial value is empty.